openapi/sled-agent.json

```diff
@@ -1464,29 +1464,6 @@
         }
       }
     },
-    "/timesync": {
```

Comment:
I wonder if there's any documentation that relies on checking the response from this endpoint in the logs to debug an NTP sync issue or something similar. I have a vague memory of something like that. If anyone remembers, that documentation should be updated. Perhaps an announcement on Matrix might be good as well? I've certainly used it before if an omicron deployment was taking too long to start.